geolocator_apple/pubspec.yaml

```
@@ -17,8 +20,9 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  
-  geolocator_platform_interface: ^2.3.2
+  # TODO: Path reference should be removed
```

nit: It is very helpful for others to explain when the path reference should be removed, similar to the comment you added at the `publish_to` tag:

```suggestion
  # TODO: Path reference should be removed when the final version of the geolocator_platform_interface is published.
```